src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedObj = {};
+
+  const sourceArray = sourceString
+    .split(';')
+    .filter((style) => style.trim() !== '');
```

Comment:
[CODE STYLE] All chained methods called on the same level should have the same indentation. This makes code much more readable.